servant/src/main/java/com/mtramin/servant/GoogleApiClientObservable.java

```diff
@@ -45,11 +45,11 @@ private GoogleApiClientObservable(Context context, GoogleApi googleApi) {
 
     static Observable<GoogleApiClient> create(Context context, GoogleApi googleApi) {
         return Observable.fromEmitter(new GoogleApiClientObservable(context, googleApi),
```

Comment:
As `Observable.fromEmitter(Action1, BackpressureMode)` has been deprecated with RxJava v1.2.7 this should use `Observable.create(Action1, BackpressureMode)` instead. (Just change `fromEmitter` to `create` and that should work)